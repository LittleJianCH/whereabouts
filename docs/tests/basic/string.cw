query (abc) {
  equation abc = "abc"
}
