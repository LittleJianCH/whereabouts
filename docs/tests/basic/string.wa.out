{ "success": true, "count": 1, "solutions": [["abc"]] }
